prime.css

```diff
@@ -0,0 +1,65 @@
+*{
+    margin:0;
+    padding:0;
+}
+
+body{
+    height: 100vh;
+}
+#container{

```

Comment:
Just after the class names and before the curly brace, give a space.

This: `#container {` instead of this: `#container{`